splice_machine/src/main/java/com/splicemachine/SpliceKryoRegistry.java

```diff
@@ -898,5 +899,14 @@ public Optional read(Kryo kryo, Input input, Class<Optional> type) {
         instance.register(TriggerDescriptorV3.class,EXTERNALIZABLE_SERIALIZER,335);
         instance.register(TriggerDescriptorV4.class,EXTERNALIZABLE_SERIALIZER,336);
         instance.register(Vector.class,110);
+        instance.register(SPSDescriptor.class,EXTERNALIZABLE_SERIALIZER,337);
+        try {
+            instance.register(Class.forName("splice.com.google.common.collect.EmptyImmutableList"), 338);
```

Comment:
Same, why Class.forName?

Reply:
EmptyImmutableList is package private and not directly accessible.

Reply:
I see, thanks!